### PR TITLE
Rebuild/fix

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mikemhenry** [comment]
@conda-forge-admin, please rerender

**Comment by mikemhenry** [comment]
Getting some issues with symbols, and it looks like they are coming from `pytorch_cluster`, which doesn't surprise me since it also doesn't  pin `pytorch` hard enough.


```
pytorch_cluster 1.5.9 py39hf39d0cb_0
------------------------------------
file name   : pytorch_cluster-1.5.9-py39hf39d0cb_0.tar.bz2
name        : pytorch_cluster
version     : 1.5.9
build       : py39hf39d0cb_0
build number: 0
size        : 823 KB
license     : MIT
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/pytorch_cluster-1.5.9-py39hf39d0cb_0.tar.bz2
md5         : 0c4f7e032f481eaa062d985f0bcbf841
timestamp   : 2022-01-12 16:33:10 UTC
dependencies: 
  - __glibc >=2.17
  - cudatoolkit 11.1|11.1.*
  - libgcc-ng >=9.4.0
  - libstdcxx-ng >=9.4.0
  - pytest
  - pytest-cov
  - python >=3.9,<3.10.0a0
  - python_abi 3.9.* *_cp39
  - pytorch >1.4.0*
  - pytorch-gpu
  - scipy
```

@hmaarrfk I'm I going down a rabit hole here? Or should I keep going through and setting up different pytorch build variants?

**Comment by hmaarrfk** [comment]
Can we try rebuilding that? Do you have Powers there?

**Comment by hmaarrfk** [comment]
This is my best attempt at pinning:
https://github.com/conda-forge/torchvision-feedstock/pull/42

I'll likely merge this soon.

**Comment by mikemhenry** [comment]
I don't have powers to merge, but I can make a PR if you are happy with the method in conda-forge/torchvision-feedstock#42

**Comment by hmaarrfk** [comment]
Why not request to be added as a maintainer. It seems that they could use help here. You can do that in a separate PR.

**Comment by mikemhenry** [comment]
Good idea!
https://github.com/conda-forge/pytorch_sparse-feedstock/pull/31

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,16 +27,13 @@ requirements:
   host:
     - python
     - pip
-    - pytorch-gpu  # [cuda_compiler_version not in (undefined, 'None')]
-    - pytorch-cpu  # [osx or cuda_compiler_version in (undefined, 'None')]
+    - pytorch {{ pytorch }} {{ torch_proc_type }}*
```
Comment (hmaarrfk) [inline recipe/meta.yaml:30]:
```suggestion
    - pytorch
    - pytorch =*={{ torch_proc_type }}*
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,16 +27,13 @@ requirements:
...
     - setuptools
     - pytest-runner
     - scipy
   run:
     - python
-    - pytorch
-    - {{ pin_compatible('pytorch-gpu') }}  # [cuda_compiler_version not in (undefined, 'None')]
-    - {{ pin_compatible('pytorch-cpu') }}  # [osx or cuda_compiler_version in (undefined, 'None')]
+    - pytorch {{ pytorch }} {{ torch_proc_type }}*
```
Comment (hmaarrfk) [inline recipe/meta.yaml:36]:
```suggestion
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Hopefully this should work. I can probably merge this in when it does.

**Comment by hmaarrfk** [comment]
ping me when you need a merge on your recipes, hopefully this model helps you build them up.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Pull request merged**